### ISSUE TITLE
Train baseline on gpu

### ISSUE THREAD

**Issue by rmattson1008** [rmattson1008/face-less#3]
Start off by training vgg on the pure LFW data. We may need some form of normalization. Logging is dependent on issue #2. 

Since VGG has been used on face data, we should be able to use hyperparameters (like learning rate, batch size) that have been proven to work in the literature.
